firmware/bootloader/bootloader.cpp

```diff
@@ -286,14 +286,18 @@ int main(void)
 		if (bootloader_ctrl == DFU_MODE) {
 			ubertooth_init();
 			run_bootloader();
+			bootloader_ctrl = 0;
+			wait(1);
+			reset();
```

Comment:
Ignore that, answered my own question - this is the reset after the firmware update.